app/components/Menu/index.js

```diff
@@ -15,13 +15,19 @@ const Wrapper = styled.div`
   display: flex;
   flex-direction: row;
   flex-wrap: nowrap;
-  justify-content: space-around;
+  justify-content: space-between;
   align-items: center;
+  padding: 1rem 4rem;
 `;
 
 const Nav = styled.ul`
   display: flex;
   flex-direction: row;
+
```

Comment:
noob question: shouldn't we try to at least implement mobile first solution, is it still valid solution within styled components ?